File: Prog II/Prog2_teste/Beginner/src/desvio_condicional/Ex_12.java
package desvio_condicional;

import java.util.Scanner;

public class Ex_12 {
	
	public static void main (String[] args) {

		Scanner sc = new Scanner(System.in);
		
		int chave;
		
		System.out.println("Digite o codigo do cargo");
		chave = sc.nextInt();
		
		switch (chave) {
		case 101:
			System.out.println("Vendedor");
			break;

		case 102:
			System.out.println("Atendente");
			break;
			
		case 103:
			System.out.println("Auxiliar Tecnico");
			break;
			
		case 104:
			System.out.println("Assistente");
			break;

		case 105:
			System.out.println("Coordenador de Grupo");
			break;
			
		case 106:
			System.out.println("Gerente");
			break;
			
		default:
			System.out.println("Invalido");
			break;
		}
		
	}

}

File: Prog1/PROG/Lista2/Nova pasta/Ex12.c
#include <stdio.h>
#include <math.h>
#include <locale.h>

int main () {
    setlocale (LC_ALL, "portuguese");
    int codigo;
    printf ("\nDigite seu código de funcionário\n");
    scanf ("%d", &codigo);
    switch (codigo) {
        case 101:
            printf ("\nCódigo de Vendedor\n");
            break;
        case 102:
            printf ("\nCódigo de Atendente\n");
            break;
        case 103:
            printf ("\nCódigo de Auxiliar Técnico\n");
            break;
        case 104:
            printf ("\nCódigo de Assistente\n");
            break;
        case 105:
            printf ("\nCódigo de Coordenador de Grupo\n");
            break;
        case 106:
            printf ("\nCódigo de Gerente\n");
            break;
        default:
            printf ("\nCódigo Inválido\n");}

    return 0;

    }
